### PR TITLE
Revert "Fixed a typo in UniversalRobot."

### PR DESCRIPTION
**Pull request by mary-doc**

### CODE REVIEW

**Comment by jaacoppi** [comment]
Why are you reverting this? Please always add a description to pull requests.

I can't test this right now, but looks like the commit that fixes a typo also includes a ^M at the end of the line, which might cause UniversalRobot to stop working. I this the reason for the revert?

**Comment by valdisvi** [comment]
There are several configuration files where DOS-style linebreaks are used (e.g. Alicia, Andy, Annie, etc.) and which work without problems.

File: espeak-ng-data/voices/!v/UniRobot [hunk 1]
```diff
@@ -1,5 +1,5 @@
 language variant
-name UniversalRobot
+name UniiversalRobot
```
Comment (mary-doc) [inline espeak-ng-data/voices/!v/UniRobot:2]: +name uniiversalRobot

**Comment by valdisvi** [comment]
This looks like trolling, see also https://github.com/ruby/ruby/pull/4310

**Pull request closed without merging**